client/src/components/Workbench/WelcomeMessages.js

```diff
@@ -32,14 +35,46 @@ const WelcomeMessages = () => {
         </Alert>
       </Route>
       <Route path={`${path}${ROUTES.DATA}`}>
-        <Alert
-          isOpen={welcomeMessages.datafiles}
-          toggle={() => onDismissWelcome('datafiles')}
-          color="secondary"
-          className="welcomeMessage"
-        >
-          This page allows you to upload and manage your files.
-        </Alert>
+        <div className="welcomeMessage">
+          <Alert
+            isOpen={welcomeMessages.datafiles}
+            toggle={() => onDismissWelcome('datafiles')}
+            color="secondary"
+          >
+            This page allows you to upload and manage your files.
+          </Alert>
+          {showWork2Message && (
+            <Alert
+              isOpen={welcomeMessages.work2}
+              toggle={() => onDismissWelcome('work2')}
+              color="warning"
+            >
+              <>
+                Notice: The Stockyard <code>/work</code> filesystem is being
+                deprecated. A new filesystem <code>/work2</code> is now
+                available and will eventually replace <code>/work</code>. During
+                the transition period, users must migrate any current data they
+                wish to retain. You can read more information about this change
+                in the{' '}
+                <a
+                  href="https://portal.tacc.utexas.edu/tutorials/stockyard-work-migration"
+                  target="_blank"
+                  rel="noreferrer"
+                >
+                  /work Migration and Transition Guide
```

Comment:
```suggestion
                  <code>/work</code> Migration and Transition Guide
```

Reply:
Hm, maybe hold off on this. Let Design answer [my inquiry](https://tacc-team.slack.com/archives/CQUA4D5KJ/p1617313138019400?thread_ts=1617313042.019100&cid=CQUA4D5KJ).

Reply:
__Update__: The suggestion is changed to match [Design request](https://tacc-team.slack.com/archives/CQUA4D5KJ/p1617314437020900?thread_ts=1617313042.019100&cid=CQUA4D5KJ).

Any concerns with this change? (technical accuracy, vagueness, etcetera)

Reply:
For this part I used verbiage straight from the email sent out from TACC User Services. If you want to diverge from it that's ok, but that's why I chose those words. I also think /work means more to a user than Stockyard.

Reply:
Cool, thanks. Please retain the original doc name form the e-mail; do not use suggestion.